src/plugins/action_update_review_status/src/databaseMethods.ts

```diff
@@ -0,0 +1,124 @@
+const databaseMethods = (DBConnect: any) => ({
+  getAssociatedReviews: async (applicationId: number) => {
+    const text = `
+    SELECT
+      review.id AS "reviewId",
+      review_assignment_id AS "reviewAssignmentId",
+      review_assignment.application_id AS "applicationId",
+      review_assignment.reviewer_id,
+      review_assignment.level,
+      review_assignment.id AS "reviewAssignmentId",
+      review_status_history.status AS "reviewStatus"
+      FROM review JOIN review_assignment
+      ON review.review_assignment_id = review_assignment.id
+      JOIN review_status_history ON review.id = review_status_history.review_id
+      WHERE review_status_history.is_current = true
+      AND review_assignment.application_id = $1
+    `
+    try {
+      const result = await DBConnect.query({ text, values: [applicationId] })
+      return result.rows
+    } catch (err) {
+      console.log(err.message)
+      throw err
+    }
+  },
+  getReviewQuestionAssignments: async (reviewAssignmentId: number) => {
+    const text = `
+    SELECT template_element_id AS "templateElementId"
+    FROM review_question_assignment
+    WHERE review_assignment_id = $1
+    `
+    try {
+      const result = await DBConnect.query({ text, values: [reviewAssignmentId] })
+      return result.rows.map(({ templateElementId }: any) => templateElementId)
+    } catch (err) {
+      console.log(err.message)
+      throw err
+    }
+  },
+  getAllReviewResponses: async (reviewId: number) => {
```

Comment:
i think from here downwards, duplicates form postresConnect

Reply:
Ah yes, I duplicated the previous Action's databaseMethods and modified -- guess I left a few behind 😊
764de8e